docs/new-architecture-library-android.md

```diff
@@ -137,10 +137,10 @@ class NativeAwesomeManager(reactContext: ReactApplicationContext?) :
 
     companion object {
         val NAME = "NativeAwesomeManager"
-            get() = Companion.field
     }
```

Comment:
Could you move the `companion object {}` block below the `getName` function?

Reply:
Done